### PR TITLE
Sticky header and footer adjustments

### PR DESCRIPTION
**Pull request by matthewhall**
Improvements to the header functionality to make it sticky on desktop after you've scrolled passed the first hero section, and the Download Report button sticky to the bottom of the page on mobile after you've scrolled passed the hero section.

Also adjusts the footer to match mdn.dev.

### CODE REVIEW

File: src/components/header.js [hunk 1]
```diff
@@ -75,6 +70,32 @@ class Header {
       document.body.classList.remove('is-header-fixed');
     }, 200);
   }
+
+  /**
+   * Handle showing the fixed button.
+   * @param {CustomEvent} event Event object.
```
Comment (joshJarr) [inline src/components/header.js]: I don't see this CustomEvent, did we remove this param from the function?

**Comment by joshJarr** [comment]
What's going on with the commit history here? We should squash this when merging.

File: src/components/header.js (hunk 1 above)
Comment (matthewhall) [reply src/components/header.js]: As in, you don't see it being passed to the method?

Comment (matthewhall) [reply src/components/header.js]: Ah, yes! Will fix.

**Pull request merged**